org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsi/participants/XSIFormatterParticipant.java

```diff
@@ -155,4 +158,58 @@ public String getCurrentLineIndent(XMLBuilder xml, XMLFormattingOptions formatti
 		return indent.toString();
 	}
 
+	@Override
+	public boolean formatAttributeValueNew(XMLFormatterDocumentNew formatterDocument,
+			int indentLevel, XMLFormattingOptions formattingOptions, DOMAttr attr,
+			List<TextEdit> edits) {
+
+		String name = attr.getName();
+		String split = formattingOptions.getXsiSchemaLocationSplit();

```

Comment:
Get the Split enum like old formatter do that and uses this enum instead of using the string.

Once yiu have the Split enum xheck if it is none